api/tests/functional-tests/backend/core/test_geometry.py

```diff
@@ -27,6 +102,52 @@ def create_clf_dataset(db: Session, dataset_name: str):
     )
 
 
+@pytest.fixture
+def create_objdet_dataset(
```

Comment:
typo in function name

Reply:
short hand for object detection, ill write it full length